dev-libs/folly/metadata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>yuchen.xie@live.com</email>
+		<name>Yuchen Xie</name>
+		<description>Proxy-Maintainer. Assign bugs to him</description>
```

Comment:
Please remove the `<description/>`. We're currently working on reducing the number of obvious descriptions in metadata.xml. The descriptions need to be read by humans and therefore block semi-automated bug assignment.